src/iexit.cpp

```diff
@@ -1,19 +1,31 @@
 /*** Copyright (c), The Regents of the University of California            ***
  *** For more information please refer to files in the COPYRIGHT directory ***/
+#include <irods/irods_exception.hpp>
+#include <irods/irods_service_account.hpp>
 #include <irods/rods.h>
 #include <irods/parseCommandLine.h>
 #include <irods/rcMisc.h>
 
+#include <cstdio>
+
 void usage( char *prog );
 
+// TODO: replace call to parseCmdLineOpt with something modern and not awful.
+//       make command line arguments more normal and boring:
+//           -y/--yes/--assume-yes: assume yes for all prompts
+//           -f/--force: allow auth file removal for svc acct
+//           -v/--verbose: increase verbosity, can be specified multiple times
+//           -h/--help/-?: show usage
+//       this may require re-implementing some of the functionality of obfRmPw()
```

Comment:
Is this required for the issue being addressed?

Reply:
No, it's more of a note for myself. I'm going to remove it once I've got an issue open for it.